core/state/statedb.go

```diff
@@ -892,9 +928,10 @@ func (s *StateDB) IntermediateRoot(deleteEmptyObjects bool) common.Hash {
 	return s.trie.Hash()
 }
 
-// Prepare sets the current transaction hash and index which are
-// used when the EVM emits new state logs.
-func (s *StateDB) Prepare(thash common.Hash, ti int) {
+// SetTxContext sets the current transaction hash and index which are
```

Comment:
sets the current transaction hash, index and accessList

Reply:
This is nit and the comment is from go-ethereum so a fix can make future conflict